pyproject.toml

```
@@ -4,3 +4,43 @@ requires = [
     "wheel"
 ]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "viur-cli"
+dynamic = ["version"]
+authors = [
+    {name = "Andreas H. Kelch", email = "ak@mausbrand.de"}
+]
+description = "Command-line interface for ViUR application maintenance."
+readme = "README.md"
+requires-python = ">=3.11"
+license = "MIT"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Operating System :: OS Independent",
+]
+dependencies = [
+    "app-server~=1.0",
+    "click~=8.1.7",
+    "pipfile-requirements~=0.3",
+    "requests~=2.0",
+    "semver~=3.0",
+    "watchdog~=6.0",
+    "safety~=3.5",
+    "viur-scriptor-api~=1.2.0",
```

This differs to the one defined in Pipfile.

```suggestion
    "viur-scriptor-api~=2.0",
```